settings/provisioner/openstack/openstack.spec

```diff
@@ -30,6 +30,12 @@ subparsers:
                       type: Value
                       help: An image id or name, on OpenStack cloud to provision the instance with. To see full list of images avaialable on the cloud, use 'glance image-list'.
                       required: yes
+            - title: ssh_user
+              options:
+                  ssh_user:
+                      type: Value
+                      help: The name of the user Ansible would use when connecting the provisioned nodes
```

Comment:
Let's just call it `username` instead of `ssh_user`.
or even better: `image-username` or `image-user` as this is a parameter of the image. (this would require changing `image` to `image-name`

No need to mention ansible in help.
```yaml
help: username to connect to all openstack nodes provisioned of the given image
```